### ISSUE TITLE
Tag releases with version (e.g. dockerfile/elasticsearch:1.3.1)

### ISSUE THREAD

**Issue by ramseykhalaf** [dockerfile/elasticsearch#11]
Would be very useful to have numbered version tags in line with elasticsearch versions. i.e.:

dockerfile/elasticsearch:1.3.1
dockerfile/elasticsearch:1.2.3

Right now I am using elasticsearch in a scala project and the client library hasn't updated itself. Which is fine, except I can't build anything except the latest version of elasticsearch, which now means I have to reference images by hashes now, not the ideal solution.

I don't know if this is possible by just adding tags on github, or by possibly tagging the hashes on dockerhub.

**Comment by solidfoxrock** [comment]
:+1: It'd be great if we can have this

**Comment by SHSE** [comment]
:+1: Please.

**Comment by pilwon** [comment]
The Docker team suggested moving images under `dockerfile` namespace to community driven official images using [this technique](https://github.com/docker-library/official-images). Tagging images is currently on hold due to this anticipated transition.

If you can volunteer to create and maintain an official image for `elasticsearch`, please go ahead then let us know so we can redirect this image to the official image.

**Comment by drazzib** [comment]
Hi all!

FTR, I've just spotted that an "official" [elasticsearch](https://registry.hub.docker.com/_/elasticsearch) image is now provided on Docker Registry with proper versionning ! Source Dockerfile is managed in [this repository](https://github.com/docker-library/elasticsearch)

Thanks to @InfoSiftr for this effort !

I think it's safe to close this issue.

**Comment by jamescarr** [comment]
Yep, been using https://registry.hub.docker.com/_/elasticsearch/ for a week
now. Love it. Thanks for the work all.
